flow-typed/npm/classnames_v2.x.x.js

```diff
@@ -0,0 +1,16 @@
+// flow-typed signature: cf6332fcf9a3398cffb131f7da90662b
+// flow-typed version: dc0ded3d57/classnames_v2.x.x/flow_>=v0.28.x
```

Comment:
This means this is a publicly available typing? Why are we committing this file?